pkg/api/content_units.go

```diff
@@ -1,9 +1,9 @@
 package api
 
 type ContentUnitListRequest struct {
-	UUID   string `param:"uuid"`    // Identifier of the repository
-	Search string `query:"search"`  // Search string based query to optionally filter-on
-	SortBy string `query:"sort_by"` // SortBy sets the sort order of the result
+	UUID   string `param:"uuid" validate:"required"`   // Identifier of the repository
+	Search string `query:"search" validate:"required"` // Search string based query to optionally filter-on
```

Comment:
search nor UUID is required here. Search is an optional filter and the UUID comes from the URL, so it's not actually passed by the user through the request body.